package.json

```diff
@@ -23,15 +23,19 @@
   },
   "homepage": "https://github.com/jeffrey20122021/workshop-api-testing-js#readme",
   "devDependencies": {
+    "@babel/eslint-parser": "^7.18.2",
     "chai": "^4.3.6",
+    "chai-subset": "^1.6.0",
     "eslint": "^8.18.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "http-status-codes": "^2.2.0",
+    "karma-chai-subset": "^2.0.1",
     "mocha": "^10.0.0",
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "md5": "^2.3.0"
```

Comment:
md5 dependency should be a dev dependency. To fix it, run: `npm i -D md5`

Reply:
i did it. Thanks